model/cluster_installation.go

```diff
@@ -124,3 +136,13 @@ func MigrateClusterInstallationResponseFromReader(reader io.Reader) (*MigrateClu
 
 	return &migrateClusterInstallationResponse, nil
 }
+
+func NewClusterInstallationStatusFromReader(reader io.Reader) (*ClusterInstallationStatus, error) {
```

Comment:
Shouldn't we get a lint error when a public function does not have a comment? cc: @gabrieljackson

Reply:
We need to configure that in linters. Looks like it's missing.

Reply:
Huh, that used to be working. We must have lost it along the way somehow.